public/docs/ts/latest/guide/template-syntax.jade

```diff
@@ -194,16 +184,15 @@ include ../_util-fns
 
   In Angular terms, an idempotent expression always returns *exactly the same thing* until
   one of its dependent values changes.
-// #enddocregion template-expressions-guidelines
-// #docregion template-expressions-guidelines-2
 :marked
   Dependent values should not change during a single turn of the event loop.
   If an idempotent expression returns a string or a number, it returns the same string or number
-  when called twice in a row. If the expression returns an object (including a `Date` or `Array`),
+  when called twice in a row. If the expression returns an object (including #{_a} `#{_Array}`),
```

Comment:
Good catch re. the article. Fixed.

Yes, I removed `Date` because the class name is `DateTime` in Dart and didn't want to define a new conditional variable just for that. I think that the point of that phrase remains clear even without Date/DateTime mentions (actually, I'm not even sure that the parenthetical remark is needed _at all_, but it is certainly ok with just mentioning Array/List IMHO).